guides/developers/event-display.md

```diff
@@ -56,42 +57,62 @@ Currently the sub managers are not big enough to be divided into multiple parts.
 
 ## [`UIManager`](../../packages/phoenix-event-display/src/managers/ui-manager/index.ts)
 
-The `UIManager` is responsible for all the UI aspects of the `EventDisplay`. This includes the Phoenix menu, dat.GUI menu, stats UI etc.
+The `UIManager` is responsible for all the UI aspects of the `EventDisplay`. This includes the Phoenix menu, dat.GUI menu, stats UI, etc.
 
 It takes care of tasks like:
 
 * Adding configuration options to Phoenix UI on loading geometry or event data.
-* Setting the UI color theme to dark or light
-* Applying clipping to geometries
-* Linking UI menu options with the `EventDisplay`
+* Setting the UI color theme to dark or light.
+* Applying clipping to geometries.
+* Linking UI menu options with the `EventDisplay`.
+* Configuring different color options for event-data.
 * etc.
 
 The `UIManager` is further divided into the following managers:
 
+* [**`ColorOptionsUI`**](../../packages/phoenix-event-display/src/managers/ui-manager/color-options.ts)  
```

Comment:
I don't think there is a need to mention the color options. It is more like a helper for the `PhoenixMenuUI`.

Reply:
Sure, removing it.